src/Components/profile/Profile.js

```diff
@@ -0,0 +1,227 @@
+              <span className="attendees">
+                <PersonIcon className={viewEvents ? "" : "icon-hide-person"} />
+                <span>{element.attendees}</span>
+                <span className="of-text">of</span>
+                <span>{element.capacity}</span>
+              </span>
+              <Button
+                variant="contained"
+                className={`button-event ${element.id}`}
+                onClick={(e) => {
+                  handleButtonEvent(
+                    e,
+                    setGoToEditEvent,
+                    setEventToEdit,
+                    eventToEdit,
+                    eventsList,
+                    setEventList
+                  );
+                }}
+              >
+                {element.stateEvent}
+              </Button>
+            </div>
+          </div>
+        );
+      }
+    });
+
+  if (signOut) {
+    return <Navigate to="/" />;
+  } else if (goToDashboard) {
+    return <Navigate to="/dashboard" />;
+  } else if (goToDetailEvent) {
+    return <Navigate to="/detailEvent" state={{ eventClicked }} />;
+  } else if (goToEditEvent) {
+    return <Navigate to="/editEvent" state={{ eventToEdit }} />;
+  } else {
+    return (
+      <div className="profile-container">
+        <section className="user-name-profile-container">
+          <div className="avatar-name-profile-container">
+            <Avatar>{textAvatar}</Avatar>
+            <span className="user-name-text-p">{userName}</span>
+            <span
+              className="arrow-container-p"
+              onClick={() => {
+                setSeeModal(!seeModal);
+              }}
+            >
+              <ArrowDropDownIcon />
+            </span>
+          </div>
+        </section>
+        <div className="user-information-profile-container">
+          <Avatar className="user-avatar">{textAvatar}</Avatar>
+          <div className="user-information-p">
+            <h4 className="user-name-p">{nameUser}</h4>
+            <p className="email-user-p">{emailUser}</p>
+            <h4 className="user-name-p">{lastNameUser}</h4>
+          </div>
+        </div>
+        <div className="dashboard-view">
+          <span>My Events</span>
+          <span>
+            <ViewModuleIcon
+              className="viewModule-icon"
+              onClick={() => {
+                setViewEvents(true);
+              }}
+            />
+            <ViewStreamIcon
+              className="viewStream-icon"
+              onClick={() => {
+                setViewEvents(false);
+              }}
+            />
+          </span>
+        </div>
+        <div
+          className={
+            viewEvents ? "dashboard-view-row" : "dashboard-view-column"
+          }
+        >
+          {displayEvents}
+        </div>
+        <ReactPaginate
+          previousLabel={"Previous"}
+          nextLabel={"Next"}
+          pageCount={pageCount}
+          onPageChange={changePage}
```

Comment:
there is no need to create a wrapper function, you can do this: `onPageChange={ ()=> changePage({ selected }) }`